include/z64skybox.h

```diff
@@ -66,6 +66,10 @@ typedef struct {
 
 extern SkyboxFile gNormalSkyFiles[];
 
+// Tests if the skybox palette for the given skybox should be written to color index 0-127.
+// If false, it should be written to color index 128-255
+#define IS_NORMAL_SKY_PALETTE_ALLOC_FRONT(skyboxIndex) ((skyboxIndex & 1) ^ ((skyboxIndex & 4) >> 2))
```

Comment:
the name `IS_NORMAL_SKY_PALETTE_ALLOC_FRONT` is mostly gibberish to me tbh but I don't really have any better ideas

Reply:
Perhaps it should be something more direct, like `IS_NORMAL_SKYBOX_COLOR_INDEX_0`?